### PR TITLE
Added Makefile

### PR DESCRIPTION
**Pull request by codeOfRobin**

### CODE REVIEW

**Comment by codeOfRobin** [comment]
I've largely referred to SwiftLint for the files.
Makefile: https://github.com/realm/SwiftLint/blob/master/Makefile
Homebrew: https://github.com/Homebrew/homebrew-core/blob/master/Formula/swiftlint.rb

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+SWIFT_BUILD_COMMAND=$(SWIFT_COMMAND) build
+SWIFT_TEST_COMMAND=$(SWIFT_COMMAND) test
+
+install:
+	xcodebuild $(XCODEFLAGS)
+	mkdir -p "$(PREFIX)/bin"
+	cp -f "$(SWIFTPLATE_EXECUTABLE)" "$(PREFIX)/bin"
+
+uninstall:
+	rm -f "$(BINARIES_FOLDER)/swiftlint"
```
Comment (JohnSundell) [inline Makefile]: This should be `swiftplate`, right? Maybe move the name `swiftplate` to a constant?

Comment (codeOfRobin) [inline Makefile]: 😂 Lol. Looks like I removed `swiftlint` from my computer.

Comment (codeOfRobin) [inline Makefile]: Making a new commit for that.

**Pull request merged**